src/components/map/map.tsx

```diff
@@ -171,9 +194,31 @@ export class Map implements GxComponent {
     }
   }
 
+  private setUserLocation({ coords }) {
+    this.userLocationCoords = `${coords.latitude}, ${coords.longitude}`;
+    this.userLocationChange.emit(this.userLocationCoords);
+    console.log("this.userLocationCoords", this.userLocationCoords);
```

Comment:
👀

Reply:
Nooo I forgot the _console.log_ debugging :´v
I'll fix that

Reply:
```console.log()```s taken out in ab4b869